style.css

```diff
@@ -500,3 +501,143 @@ header {
   background: #fff;
   color: black;
 }
+
+@media (min-width: 764px) {
```

Comment:
- [ ] The recommended media query for the desktop version is between `768px` and `1440px`. Kindly change the media query you are using currently. 👍